### ISSUE TITLE
How much does the loss value fall? and the network is converge.

### ISSUE THREAD

**Issue by hewumars** [umich-vl/CornerNet#16]
How much does the loss value fall? and the network is converge.

**Comment by hewumars** [comment]
I use 33000 images,trained 300k its,but it is not converge. Why?

**Comment by hewumars** [comment]
@heilaw

**Comment by heilaw** [comment]
Can you provide more details about your dataset or training settings? They may help locate the issue.
